application/controller/musics_controller.go

```diff
@@ -0,0 +1,30 @@
+package controller
+
+import (
+	"github.com/Nexters/myply/domain/service"
+	"github.com/gofiber/fiber/v2"
+)
+
```

Comment:
https://github.com/Nexters/myply/pull/40/files#r928938915
This is a continuation of the comment above.

I think you need wire.newset(NewController) in here to create musicController provider( create a new instance method)

Reply:
nit: How about music.go? If you prefer *_controller.go, thats nice too!